flybywiresim/a32nx
# feat: EFB Dashboard Weather Info With Full METAR

## Summary of Changes
This PR adds the full METAR text to the flyPad dashboard weather widget. 
For this some of the layout had to be changed and a scrollbar added. 
It also remembers the chosen format between sim starts separately for origin and destination. 

## Screenshots (if necessary)
https://user-images.githubusercontent.com/16833201/149624354-6f20e4a1-d181-4b56-b5d7-072327b804ef.mp4

Discord username (if different from GitHub): Cdr_Maverick#6475

## Testing instructions

- Start anywhere
- Load EFB from simbrief or manually using the weather ICAO input fields. 
- Use the ICAO field first, before loading from simbrief. Make sure the "FROM SIMBRIEF" overwrites the manually entered ICAO. 
- Check various examples and watch for layout issues. 
- Check that it remembers the chosen format between sim starts separately for origin and destination. 


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
